sharepoint/sharepoint-ps/sharepoint-online/Get-SPOContentSecurityPolicy.md

```diff
@@ -0,0 +1,37 @@
+---
+external help file:
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spocontentsecuritypolicy
+applicable: SharePoint Online
+title: Get-SPOContentSecurityPolicy
+schema: 2.0.0
+author: jaobrie
+ms.author: jaobrie
+manager: ryannak
+ms.reviewer:
+---
+
+# Get-SPOContentSecurityPolicy
+
+## SYNOPSIS
+
+Returns the all entries in the current Content Security Policy configuration.
```

Comment:
I would add: "in multi geo environments, this command returns all the entries related to the current geo" (or something on that line)